src/core/grade_notifier.py

```diff
@@ -288,7 +289,7 @@ def start_notifier():
             if result != None \
             else None
         if changelog is not None:
-            message = create_text_message(changelog)
+            message = create_text_message(changelog, user.get_omitgrade())
```

Comment:
maybe the method name could be: user.requestedGradeOmission. 
Reads better I feel.